src/content/advanced/gitops/apps/add_appcr.md

```diff
@@ -0,0 +1,192 @@
+- An example of a directly configured App (with configuration): an [app that uses a configuration ConfigMap](https://github.com/giantswarm/gitops-template/tree/main//management-clusters/MC_NAME/organizations/ORG_NAME/workload-clusters/WC_NAME/apps/nginx-ingress-controller/)
+- An example of an App created from an App Template is available in [WC_NAME/apps/nginx-from-template](https://github.com/giantswarm/gitops-template/tree/main//management-clusters/MC_NAME/organizations/ORG_NAME/workload-clusters/WC_NAME/apps/nginx-from-template/).
+
+## Common steps
+
+Please follow these steps when installing an App directly as well as using App Template.
+
+### Export environment variables
+
+__Note__: Management Cluster codename, Organization name, Workload Cluster name and several App-related values are needed in multiple places across this instruction, the least error prone way of providing them is by exporting them as environment variables:
+
+```nohighlight
+export MC_NAME=CODENAME
+export ORG_NAME=ORGANIZATION
+export WC_NAME=CLUSTER_NAME
+export APP_NAME="${WC_NAME}-APP_NAME"
+```
+
+### Setting up directory tree structure for managing apps
+
+1. Go to the `apps` directory:
+
+    ```nohighlight
+    cd management-clusters/${MC_NAME}/organizations/${ORG_NAME}/workload-clusters/${WC_NAME}/mapi/apps
+    ```
+
+1. Create new directory with a name corresponding to the App name:
+
+    ```nohighlight
+    mkdir ${APP_NAME}
+    ```
+
+## Adding App directly
+
+1. Set remaining environment variables
+
+    ```nohighlight
+    export APP_VERSION=APP_VERSION
+    export APP_CATALOG=APP_CATALOG
+    export APP_NAMESPACE=APP_NAMESPACE
+    # OPTIONAL
+    export APP_USER_VALUES=CONFIGMAP_OR_SECRET_PATH
+    ```
+
+1. Go to the newly created directory and use [the kubectl-gs plugin](https://github.com/giantswarm/kubectl-gs) to generate the [App CR](https://docs.giantswarm.io/ui-api/kubectl-gs/template-app/):
+
+    ```nohighlight
+    cd ${APP_NAME}/
+    kubectl gs template app \
+    --app-name "\${cluster_name}-${APP_NAME}" \
+    --catalog ${APP_CATALOG} \
+    --cluster ${WC_NAME} \
+    --name ${APP_NAME} \
+    --namespace ${APP_NAMESPACE} \
+    --version ${APP_VERSION} > appcr.yaml
+    ```
+
+__Note__: you can optionally configure App with the user-provided values by adding the below flags to the previous command:
+
+    ```nohighlight
+    --user-configmap ${APP_USER_VALUES}
+    --user-secret ${APP_USER_VALUES}
+    ```
+
+__Note__: We're including `${cluster_name}` in the app name to avoid a problem when two
+    or more clusters in the same organization want to deploy the same app with its
+    default name.
+
+    Reference [the App Configuration](https://docs.giantswarm.io/app-platform/app-configuration/) for more details of how
+    to properly create respective ConfigMaps or Secrets.
```

Comment:
```suggestion
__Note__: We're including `${cluster_name}` in the app name to avoid a problem when two or more clusters in the same organization want to deploy the same app with its default name.

Reference [the App Configuration](https://docs.giantswarm.io/app-platform/app-configuration/) for more details on how to properly create respective ConfigMaps or Secrets.
```